lib/Alien/Build/MM.pm

```diff
@@ -422,7 +422,11 @@ sub mm_install
     $mm->SUPER::install(@rest);
   };
 
-  "install :: alien_clean_install\n\n$section";
+  return
+      ".NOTPARALLEL : install\n\n"
+    . ".NO_PARALLEL : install\n\n"
```

Comment:
```suggestion
    . ".NO_PARALLEL :\n\n"
```
I think `.NO_PARALLEL` is the same, I _think_ the only place it might matter is NetBSD or older versions of FreeBSD.  It looks like recent versions of FreeBSD support `.NOTPARALLEL`:
https://www.freebsd.org/cgi/man.cgi?make(1)

Reply:
Done.  So far as I can tell, dmake has .NO_PARALLEL, but it's simpler to not specify any deps.
https://docs.oracle.com/cd/E19422-01/819-3697/dmake.html